source/agora/test/Base.d

```diff
@@ -1752,13 +1752,13 @@ public struct TestConf
     bool configure_network = true;
 
     /// the delay between request retries
-    Duration retry_delay = 100.msecs;
+    Duration retry_delay = 500.msecs;
```

Comment:
Those are slightly different though. We know that the nodes are in memory so it's going to be insanely faster.
However, increasing to 500 msecs (and reducing retries) is actually a good idea, so 👍